src/components/Home.js

```diff
@@ -47,20 +81,14 @@ function Home() {
       `}
     </style>
     <div class="home-body">
-      <header className="App-header-home">
-          <img src={extension} className="App-logo-home" alt="logo" />
-          <div id="slogan">
-            <TextLoop>
-                <span>Support <span id="smile">Global Health</span></span>
-                <span>Stand with <span id="smile">Racial Justice</span></span>
-                <span>Provide <span id="smile">Water &amp; Food Security</span></span>
-                <span>Empower <span id="smile">Women</span></span>
-            </TextLoop>{" "}
-            <div className="subtitle-home">Donate <span id="emphasize">without spending extra</span> when shopping online.</div>
-            <Button variant="outline-secondary btn-round-header" size="lg" href="https://tiny.cc/soulsmile-extension">Add to chrome</Button>{' '}
-          </div><br/>       
-      </header>
-      <h3 id="home-story-header"><b>Meet Olu and Mily</b></h3>
+
+      {/* TFAC Ternary Operator 1 */}
+      {collab ? <div><AffiliateBanner collab={collab}/></div> : null }
```

Comment:
Instead of null I believe you could also just put in <></> for an empty element :)

Reply:
Updated!